modules/gapi/src/api/freetype_mask_creator.hpp

```diff
@@ -113,14 +104,22 @@ class FreeTypeBitmaskCreator : public IBitmaskCreator
 
     virtual ~FreeTypeBitmaskCreator() override
     {
-        FT_Done_Face(m_face);
-        FT_Done_FreeType(m_library);
+        GAPI_Assert(!FT_Done_Face(m_face));
+        GAPI_Assert(!FT_Done_FreeType(m_library));
     }
 
 private:
     FT_Library    m_library;
     FT_Face       m_face;
 
+    virtual void init() override
+    {
+        GAPI_Assert(!FT_Init_FreeType(&m_library));
+        GAPI_Assert(!FT_New_Face(m_library, m_font_path.c_str(), 0, &m_face));
+    }
```

Comment:
Extra init is unnecessary here

Reply:
I mean, keep doing stuff in the constructor but more carefully

Reply:
Fixed